lib/sql.js

```diff
@@ -690,9 +690,24 @@ SQLConnector.prototype.update = function(model, where, data, options, cb) {
  */
 SQLConnector.prototype._replace = function(model, where, data, options, cb) {
   var stmt = this.buildReplace(model, where, data, options);
-  this.execute(stmt.sql, stmt.params, options, cb);
+  this.execute(stmt.sql, stmt.params, options, function(err, info) {
+    if (err) return cb(err);
+    if (info[0]) info = info.shift();
```

Comment:
Why is this needed? IMO, the API contract between the connectors and the juggler should make it clear what is the expected `info` object to be returned by `_replace` call.

Also as I commented in https://github.com/strongloop/loopback-connector-postgresql/pull/203, there should be tests in the shared test suite to verify that each connector returns an `info` object in the expected shape.